src/presentation/validator/customer-validator.js

```diff
@@ -0,0 +1,31 @@
+import { body } from 'express-validator'
+import { cpf } from 'cpf-cnpj-validator';
+import isValidCep from '@brazilian-utils/is-valid-cep';
+
+var cpfMaxMin = ({ 'max': 14, 'min': 11 });
+var phoneMaxMin = ({ 'max': 14, 'min': 11 });
```

Comment:
Se o seu max for considerando o número de telefone com toda formatação, acho que o max poderia ser 15 pois:
(XX) XXXXX XXXX
 11 números + 2 parenteses + 2 espaços.

Reply:
corrigido